test/e2e/common/node/pod_resize.go

```diff
@@ -1012,6 +1013,565 @@ func doPodResizeErrorTests(f *framework.Framework) {
 	}
 }
 
+func doConsecutivePodResizeTests(f *framework.Framework) {
+	var podClient *e2epod.PodClient
+	ginkgo.BeforeEach(func() {
+		podClient = e2epod.NewPodClient(f)
+	})
```

Comment:
nit: Is it possible to initialize `podClient` inside each testcase? I refactored this for other tests in my PR #128143.